### PR TITLE
Move point clouds to benchmarks/utils

### PR DESCRIPTION
**Pull request by aprokop**
- Move point clouds to `benchmark/utils`
- Move the code itself into `ArborXBenchmark` namespace
- Rename files to camel case

### CODE REVIEW

File: benchmarks/bvh_driver/CMakeLists.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
-set(POINT_CLOUDS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/benchmarks/point_clouds)
+set(BENCHMARKS_UTILS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/benchmarks/utils)
```
Comment (dalg24) [inline benchmarks/bvh_driver/CMakeLists.txt:1]:
```suggestion
set(ARBORX_BENCHMARKS_UTILS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/benchmarks/utils)
```
to be on the safe side

**Review by dalg24 — COMMENTED** [review]
I'd be tempted to say rename the files to `ArborXBenchmark_PascalCaseName.hpp`

File: benchmarks/bvh_driver/CMakeLists.txt (hunk 1 above)
Comment (dalg24) [reply benchmarks/bvh_driver/CMakeLists.txt:1]: Actually should we say BENCHMARK[S] without the s?

Comment (aprokop) [reply benchmarks/bvh_driver/CMakeLists.txt:1]: > Actually should we say BENCHMARK[S] without the s?

No strong opinion.

**Comment by aprokop** [comment]
> I'd be tempted to say rename the files to `ArborXBenchmark_PascalCaseName.hpp`

I thought about it and would be fine, just wanted to keep the diff down.

**Comment by aprokop** [comment]
> I'd be tempted to say rename the files to `ArborXBenchmark_PascalCaseName.hpp`

Done.

**Review by dalg24 — APPROVED** [review]

**Pull request merged**